api/package.json

```diff
@@ -13,6 +13,6 @@
   },
   "scripts": {
     "toc": "doctoc --github --maxlevel 2 README.md",
-    "run-watch": "TZ=UTC nodemon --inspect=0.0.0.0:9229 --ignore 'build/static' --ignore 'build/public' --watch ./ --watch '../shared-libs/**/src/**' server.js -- --allow-cors"
+    "run-watch": "TZ=UTC nodemon -e js,json,html,properties --inspect=0.0.0.0:9229 --ignore 'build/static' --ignore 'build/public' --watch ./ --watch '../shared-libs/**/src/**' server.js -- --allow-cors"
```

Comment:
Did you intend for this update to remain?

Reply:
We do want to keep this! Apparently nodemon [does not watch html files by default](https://medic.slack.com/archives/C07U4E31K40/p1738686544487489?thread_ts=1738671082.341249&cid=C07U4E31K40).